pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,190 @@
+>   - `[inspector]`: optional Avo Inspector object. If you use Avo Inspector pass it here to make the Avo functions automatically report the invocations to the Avo Inspector. 
+>   - `[mobileDebugger]`: React Native specific and optional Avo Debugger instance. Pass it to make the Avo functions automatically show the functions calls and all the errors in the visual debugger. Check <a target="\_blank" rel="noopener noreferrer" href="https://github.com/avohq/react-native-analytics-debugger">React Native mobile debugger repo</a> to learn more about it.
+>
+> `systemProperties`: an object where each field represents a system propety in your tracking plan. 
+> When you define system properties in your Avo workspace you set name and type - the fileds of this object are named the same as system propertyies, in lower camel case, and you should provide corresponding types.
+>
+> `destinationOptions`: `{[segment], [amplitude], [mixpanel]}`
+>   - `mixpanel`: optional object, if you use Mixpanel destination managed by Avo, this object will be passed to `mixpanel.init(apiKey, options)` as the second parameter, `options`
+>   - `amplitude`: optional object, if you use Amplitude destination managed by Avo, this object will be passed to `amplitude.init(apiKey, null, options)` as the third parameter, `options`
+>   - `segment`: optional object, if you use Segment destination managed by Avo, this object will be passed to `analytics.load(apiKey, options)` as the second parameter, `options`
+>
+> `customDestination`: required if given source has a custom destination set up in Avo. Otherwise ignored.
+>
+> Read more about custom destinations <Link passHref href="/implementation/strat-custom-destination"><a>here</a></Link>.
+>
+> Find examples of JavaScript custom destinations for each analytics destination in <Link passHref href="/implementation/destinations"><a>this section</a></Link>.
+
+##### setAvoLogger
+
+```javascript
+Avo.setAvoLogger(avoLogger)
+```
+
+Separarte method to set custom logger
+
+> **Arguments**
+>
+> `avoLogger`: custom implementation of the logger. Can be used to disable logs. <Link passHref href="/implementation/custom-loggers"><a>More about custom loggers</a></Link>  
+
+##### setSystemProperties
+
+```javascript
+Avo.setSystemProperties(systemProperties)
+```
+
+A method to update system properties. If you provide undefined values here corresponding properties won't be updated
+
+>**Arguments**
+>
+>`systemProperties`: an object where each field represents a system propety in your tracking plan. 
+>When you define system properties in your Avo workspace you set name and type - the fileds of this object are named the same as system propertyies, in lower camel case, and you should provide corresponding types.
+
+##### Event tracking functions
+
+```javascript
+Avo.yourEventName(properties: { [anonymousUserId_], [userId_], [segmentContext_], [eventProperty0], [eventProperty1], ..., [userProperty0], [userProperty1], ... })
```

Comment:
I think we should not mix soruce or destination specific parameters in here. Keep this generic enough so people realize this is totally dependent on their workspace
```suggestion
Avo.yourEventName(properties: { [eventProperty1], [eventProperty2], ..., [userProperty1], [userProperty2], ... })
```